pwnlib/util/proc.py

```diff
@@ -99,12 +99,11 @@ def name(pid):
         Name of process as listed in ``/proc/<pid>/status``.
 
     Example:
-        >>> pid = 1
-        >>> n = name(pid)
-        >>> n
-        'systemd'
-        >>> pid in pidof(n)
-        True
+        >>> from pwnlib import context
+        >>> from pwnlib.tubes.process import process
```

Comment:
Do you think that this should be hidden in this file instead?
https://github.com/Gallopsled/pwntools/blob/daa87d64384ff670c755c029869987b28fdf2472/docs/source/util/proc.rst#L1-L5
While looking at its history I noticed this commit, which looks like just what we would need:
https://github.com/Gallopsled/pwntools/commit/dbf5145bbf88cb627251725f3f032a0dca9e96f7
I wonder why it was reverted (just for the coverage to increase?):
https://github.com/Gallopsled/pwntools/commit/3b65f8775a046cc1db1eacd8f701cb36853fd951

Reply:
Regarding that specific reversion, I expect it was simply overlooked in a merge conflict. Re-applying dbf5145b is fine with me.

Reply:
Separately, for our testing setup I don't think you need to use `context.quiet`.  If you look in `docs/conf.py`, we set the log level to `ERROR` for testing purposes.